fuel-async/fuel-reactor/src/main/kotlin/fuel/ReactorExtension.kt

```diff
@@ -0,0 +1,21 @@
+package fuel
+
+import okhttp3.Call
+import okhttp3.Callback
+import okhttp3.Response
+import reactor.core.publisher.Mono
+import java.io.IOException
+
+public fun Call.toMono(): Mono<Response> =
+    Mono.create {
+        enqueue(object : Callback {
```

Comment:
Hmm I think we should prefer `execute` over `enqueue`

Reply:
I kinda surprised to see it works. :)